platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/Round.java

```diff
@@ -42,6 +43,7 @@ public interface Round extends Iterable<ConsensusEvent> {
      * @return an iterator of consensus events
      */
     @Override
+    @NonNull
```

Comment:
Annotated `@NonNull`, but overridden methods are not annotated

Reply:
fixed